maths/combinations.py

```diff
@@ -1,19 +1,66 @@
+"""
+This function will return the total number of different combinations of
+k length which can be made from n values.
+
+Wikipedia page here: https://en.wikipedia.org/wiki/Combination
+
+For doctests, run: `python3 -m doctest -v combinations.py`
+
+Run `python3 combinations.py` for print statements which demonstrate use
+cases for this function.
```

Comment:
```suggestion
https://en.wikipedia.org/wiki/Combination
```

Reply:
So it's better to take out the description at the start?